README.md

```diff
@@ -23,7 +23,8 @@ go-gerrit is a [Go](https://golang.org/) client library for accessing the [Gerri
 
 ## Installation
 
-go-gerrit requires Go version 1.10 or greater.
+go-gerrit follows the [Go Release Policy](https://golang.org/doc/devel/release.html#policy).
+This means, we support the current + two previous Go version releases.
```

Comment:
Replace “current + two previous” with “current + previous”. Go supports two major releases at a time, not three.

Reply:
True. Misread the policy. Thanks @dmitshur!